### PR TITLE
chore: minor fixes to avoid errors on tutorial

### PR DESCRIPTION
**Pull request by isp-uetsuki**

### CODE REVIEW

File: docs/tutorials/create_architecture.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ path:
 
 ```python
 arch.add_path('target_path', path)
+! rm ./architecture.yaml
 arch.export('./architecture.yaml')
```
Comment (takam5f2) [inline docs/tutorials/create_architecture.md:114]: How about replacing this line by the following code?
```suggestion
arch.export('./architecture.yaml', force=True)
```

**Review by takam5f2 — CHANGES_REQUESTED** [review]
@isp-uetsuki  
Thank you for this PR. Could you check one comment?

**Comment by takam5f2** [comment]
Adding "documentation" label to this PR, deploy-docs action will generate documentation for this PR. Best,

File: docs/tutorials/create_architecture.md (hunk 1 above)
Comment (isp-uetsuki) [reply docs/tutorials/create_architecture.md]: Thank you for your review. I've commited your suggestion.

**Review by takam5f2 — APPROVED** [review]
LGTM

**Pull request merged**